template/css/utilities.css

```diff
@@ -23,8 +23,12 @@
   background: var(--primary-color);
   color: #000000;
 }
+
 .btn-secondary {
-  background: var(--bg-primary);
+  margin: 5% auto;
```

Comment:
Hello, @winniffy can you use `px` for the margin-top and bottom instead? 
Using percentages will cause the button to move when the screen is smaller.

```css
.btn-secondary {
 margin: 5px 0;
}
```